docs/_content/standaard/catalogi/index.md

```diff
@@ -118,6 +118,32 @@ parametrisering is aan validatieregels onderhevig:
     * MOET een waarde hebben indien de afleidingswijze `termijn` is
     * MAG GEEN waarde hebben in de andere gevallen
 
+#### Concepten
+
+De resources `Zaaktype`, `InformatieObjecttype` en `Besluittype` bevatten het veld `concept`,
+indien dit veld aangemerkt is als `true`, dan betreft het een niet-definitieve versie van
+het object. Deze versie mag niet buiten de Catalogi API gebruikt worden.
```

Comment:
Ik zou eerder spreken van een objecttype. Dan is duidelijker dat er geen objecten (bv zaken) van een objectype (bv zaaktype) aangemaakt mogen worden wanneer het objectytpe (zaaktype) concept=true bevat.

Reply:
aangepast (dank aan Steven)